docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/bias_quantification.ipynb

```diff
@@ -0,0 +1,1387 @@
+{
```

Comment:
<p>A few notes on this diagram:</p><p> </p><ul><li>I don't find the pre- and post-intervention bubbles helpful in thinking through this. In general, causal diagrams already imply the passage of time (a causal arrow must take place over time).</li><li>As above, I don't think we need O1 and O2 in this diagram, especially since they are not included in the functions below.</li><li>I don't think post-intervention R1 adds anything here, since we are focused here on the correlations between <em>two</em> risks.</li></ul><p>The isolated form of this diagram, I think, is R1 -&gt; intervention -&gt; R2 with a correlation between R1 and R2. In that situation the correlation between R1 and R2 matters to the result (as long is it is downstream of R2).</p><p> </p><p><strong>And</strong>, let's add a note that in this situation, if there is non-zero baseline intervention coverage, determining the rho to initialize with in order to validate to the real-life correlation is hard. I believe it needs to be done using something like our methods for mediation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>